.github/workflows/update-changeset-prs.yaml

```diff
@@ -6,20 +6,43 @@ on:
       - 'changeset-release/**'
 
 jobs:
+  prev-commit:
+    name: Verify previous commit
+    runs-on: ubuntu-latest
+    outputs:
+      is-valid: ${{ steps.prev-commit.outputs.is-valid }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+        with:
+          # Fetch the latest two commits
+          fetch-depth: 2
+
+      # We do not want an infinite loop; therefore, we are checking if the previous
+      # commit was created by this action.
+      - name: Get previous commit message
+        id: prev-commit
+        run: |
+          echo "is-valid=$(if [[ $(git log -1 --pretty=%B HEAD^) != 'chore: update yarn.lock' ]]; then echo 'true'; else echo 'false'; fi)" >> $GITHUB_OUTPUT
```

Comment:
bash logic looks OK, though I don't why you're checking the previous commit. should this not be the latest one?

Reply:
```suggestion
          # check the latest commit message in this branch for a yarn lock update message
          echo "is-valid=$(if [[ $(git log -1 --pretty=%B HEAD) != 'chore: update yarn.lock' ]]; then echo 'true'; else echo 'false'; fi)" >> $GITHUB_OUTPUT
```